docs/content/manual/_incoming/3003-mark-pci-experimental-enable-warning.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Mark pcidevices-controller add-on as experimental and add warning when enable devices
+---
+
+* Related issues: [#3003](https://github.com/harvester/harvester/issues/3003) [FEATURE] Mark pcidevices-controller add-on as experimental and add warning when enable devices
```

Comment:
This issue marked `not-require/test-plan`.

Reply:
The initial idea suppose the warning message deserve a check as a safety protection mechanism. 
After further consideration, I think this warning would not quite need to be changed. Thus I think to remove this case.